bin/capsule-certs-generate

```diff
@@ -45,11 +45,20 @@ CONFIG_CONTENT = <<EOF
 EOF
 
 config_file = temp_file('capsule-certs', CONFIG_CONTENT)
+
+# generate our parser cache and add it to the config file
+parser_cache = temp_file('capsule-certs.parser_cache', `kafo-export-params -c #{config_file.path} -f parsercache --no-parser-cache`)
```

Comment:
I like Option 1/ more. But it would require not to mix the real scenarios with capsule-certs-generate configuration. How about to move the current content of `config` to `scenarios.d` and `capsule-certs-generate.yaml` keep in a `config`. Then `scenarios.d` would be installed to `/etc/foreman-installer/scenarios.d` and config to `/etc/foreman-installer-katello/` or better to `/usr/share/foreman-installer-katello/config/`.